File: 10/10.py
from math import ceil, sqrt

def is_prime(n: int) -> bool:
	if n == 2: return True

	for i in range(2, ceil(sqrt(n))+1):
		if n % i == 0: return False

	return True

sum_primes: int = 2
for i in range(3, 2_000_000, 2):
	if is_prime(i):
		sum_primes += i


print(sum_primes)

File: 10/10.cpp
#include <math.h>
#include <iostream>
using namespace std;

bool is_prime(long long n) {
	if (n == 2) return true;
	for (long long i=2; i <= ceil(sqrt(n)); i++) {
		if (n%i == 0)	return false;
	}
	return true;
}

int main() {
	long long sum=2;
	for (long long i=3; i < 2000000; i += 2) {
		if (is_prime(i)) sum += i;
	}
	cout << sum << endl;
	return 0;
}
